DesktopModules/Vanjaro/UXManager/Extensions/Apps/Icon/Factories/AppFactory.cs

```diff
@@ -89,7 +90,7 @@ internal static string GetAccessRoles(UserInfo UserInfo)
                 AccessRoles.Add("admin");
             }
 
-            if (TabPermissionController.HasTabPermission("EDIT") || !Editor.Options.EditPage)
+            if (TabPermissionController.HasTabPermission("EDIT") || (HttpContext.Current.Request.Cookies["IsVjEditor"] != null && !string.IsNullOrEmpty(HttpContext.Current.Request.Cookies["IsVjEditor"].Value)))
```

Comment:
The following should be a common method if it's being used at multiple places. 

(HttpContext.Current.Request.Cookies["IsVjEditor"] != null && !string.IsNullOrEmpty(HttpContext.Current.Request.Cookies["IsVjEditor"].Value))